paxsenix/src/main/kotlin/com/metrolist/paxsenix/Paxsenix.kt

```
@@ -0,0 +1,463 @@
+                val centiseconds = (timeMs % 1000) / 10
+
+                val agent = when {
+                    line.background -> "{bg}"
+                    line.oppositeTurn -> "{agent:v2}"
+                    else -> "{agent:v1}"
+                }
+
+                val lineText = line.text.joinToString(" ") { it.text }
+
+                if (lineText.isNotBlank()) {
+                    appendLine(String.format("[%02d:%02d.%02d]%s%s", minutes, seconds, centiseconds, agent, lineText))
```

detekt (1.23.8)

[warning] String.format("[%02d:%02d.%02d]%s%s", minutes, seconds, centiseconds, agent, lineText) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)